### ISSUE TITLE
Mounting all discs of a multi-disc game at once?

### ISSUE THREAD

**Issue by Vlek** [schellingb/dosbox-pure#349]
Is it possible to mount all of the discs simultaneously instead of having to unmount and mount the second, third, and so on disc images?

**Comment by PoloniumRain** [comment]
I'm not sure about that. But if you don't already know, you can also change discs this way: 

`Quick Menu > Disc Control > Eject Disc > Current Disc Index > choose a disc > Insert Disc`

All the game discs should already be listed inside 'Current Disc Index'. You can also configure RetroArch hotkeys to change discs: 

`Settings > Input > HotKeys`

But it would be nice if DOSBox-Pure had a built in and quicker way of doing this, like with just one or two key/button presses.

**Comment by toniosj** [comment]
> Is it possible to mount all of the discs simultaneously instead of having to unmount and mount the second, third, and so on disc images?

@Vlek Yes, it's possible. Here you are an example, using this command inside dosbox.conf of Last Express game. This mount automatically three discs.

imgmount d ".\lexpress\cd\disk1.iso" ".\lexpress\cd\disk2.iso" ".\lexpress\cd\disk3.iso" -t cdrom
